feed/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Explor the newest adventure posts" />
+    <title>Feed | Xplor</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500&display=swap" rel="stylesheet" />
+
+    <link rel="stylesheet" href="../dist/css/index.css" />
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css" />
+
+    <link rel="icon" type="image/x-icon" href="/images/logo/xplor_logo-light.png" />
+    <link rel="icon" href="data:;base64,=" />
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+    <script defer src="../js/script.js"></script>
+  </head>
+  <body class="bg-primary text-white min-vh-100 d-flex flex-column">
+    <header class="bg-primary sticky-top p-1">
+      <nav class="navbar navbar-expand-sm bg-primary" data-bs-theme="dark">
+        <div class="container-fluid">
+          <a href="#" class="navbar-brand">
```

Comment:
```suggestion
          <a href="/" class="navbar-brand">
```

This should link to your homepage.

Reply:
Even when the homepage is the login/signup page? I was thinking that the link then should go to the feed page as the homepage when you are in the application.